templates/index.html

```diff
@@ -21,17 +28,23 @@ <h2>Items</h2>
       <ul class="list-group mb-4">
         {% for item in items %}
           <li class="list-group-item">
-            <b>{{ item.title }}</b>: {{ item.status }}
-            <div class="float-right">
-              <form action="{{ url_for('delete_item', id=item.id) }}" method="post" class="d-inline">
-                <button type="submit" class="btn btn-danger mr-1">Delete</button>
-              </form>
-              {% if item.status == 'Not Started' %}
-                <form action="{{ url_for('mark_item_as_complete', id=item.id) }}" method="post" class="d-inline">
-                  <button type="submit" class="btn btn-success">Mark as complete</button>
+            <div class="row">
+              <span class="col"><b>{{ item.title }}</b>: {{ item.status }}</span>
+              <div class="col-auto">
+                <form action="{{ url_for('delete_item', id=item.id) }}" method="post" class="d-inline">
+                  <button type="submit" class="btn btn-danger mr-1">Delete</button>
                 </form>
-              {% endif %}
+                {% if item.status == 'Not Started' %}
+                  <form action="{{ url_for('mark_item_as_complete', id=item.id) }}" method="post" class="d-inline">
+                    <button type="submit" class="btn btn-success">Mark as complete</button>
+                  </form>
+                {% endif %}
+              </div>
             </div>
+            
+            {% if item.description != '' %}
```

Comment:
I don't believe wrapping this in an `if` block actually affects the display of the page at all (feel free to correct me here!)